modules/client/src/test/java/org/apache/ignite/client/ReconnectTest.java

```diff
@@ -43,7 +43,7 @@ void tearDown() throws Exception {
     @Test
     public void clientReconnectsToAnotherAddressOnNodeFail() throws Exception {
         FakeIgnite ignite1 = new FakeIgnite();
-        ignite1.tables().createTable("t", c -> c.changeName("t"));
+        ignite1.tables().createTable("t", c -> {});
```

Comment:
Maybe add a consumer doing nothing to **ConfigurationTestUtils**?

Reply:
Why? =)

Reply:
Never use autoformat for entire file!